handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -538,18 +548,14 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         }
         try {
             wrap(ctx, false);
-        } catch (Throwable cause) {
-            // Fail pending writes.
-            pendingUnencryptedWrites.removeAndFailAll(cause);
```

Comment:
@Scottmitch so you want me to catch Throwable and rethrow ? I am not strong here...